PolyEngine/RenderingDevice/OpenGL/Src/GLShaderProgram.cpp

```diff
@@ -149,6 +149,10 @@ void GLShaderProgram::SetUniform(const String& name, int val)
 		HEAVY_ASSERTE(it->second.TypeName == "int" || it->second.TypeName == "sampler2D", "Invalid uniform type!");
 		glUniform1i(it->second.Location, val);
 	}
+	// else
+	// {
+	// 	gConsole.LogWarning("GLShaderProgram::SetUniform(): Unregistered Uniform: {}", name);
+	// }
```

Comment:
Why are we commenting this out?

Reply:
I've spent some time investigating why uniforms in shaders are not updating. It was because uniforms were not registered in program. Mistakes like that should be logged but those lines are called several times each mesh resulting in massive log spam. Logging it only once on first attempt to set unregistered uniform would be appropriate.